packages/frontend/src/components/DashboardFilter/Filter.tsx

```diff
@@ -198,7 +198,7 @@ const Filter: FC<Props> = ({
                                 </Text>
                             </Tooltip>
                             <Text fw={400} span>
-                                {filterRule?.disabled ? (
+                                {!filterRule?.values?.length ? (
```

Comment:
why can't we use `disabled` here? 🤔

Reply:
Good question. I think the (somewhat unsatisfying) answer is that I did this first before fixing it so that disabled and length === 0 are always the same. I think I can put this back now. Nice catch.

Reply:
Done